Python/Color_Picker/colors.csv

```diff
@@ -0,0 +1,865 @@
+air_force_blue_raf,"Air Force Blue (Raf)",#5d8aa8,93,138,168
```

Comment:
Do we need this CSV file here?

Reply:
Yes I have compared the color from the csv file.